.gitignore

```diff
@@ -13,7 +13,7 @@
 !/log/.keep
 !/tmp/.keep
 
-database.yml
+application.yml
 doc/
 *.swp
 *~
```

Comment:
@EugeniaM Please add to gitignore your folder with uploads and remove your uploads from git index